packages/oidc-client-service-worker/src/OidcServiceWorker.ts

```diff
@@ -322,13 +335,16 @@ const handleMessage = async (event: ExtendableMessageEvent) => {
 		const convertAllRequestsToCorsExceptNavigate = Array.isArray(trustedDomain)
 			? false
 			: trustedDomain.convertAllRequestsToCorsExceptNavigate;
+		const allowMultiTabLogin = Array.isArray(trustedDomain)
```

Comment:
It may have a bug, demo default behaviors seems like allowMultiTabLogin=true

Reply:
Not sure what do you mean 🤔. If `trustedDomain` is an array, then `allowMultiTabLogin` will be false. It will be true only if `trustedDomain` is an object with `allowMultiTabLogin` explicitly set to true. Analogically to `showAccessToken`.
I have checked and it looks like it actually defaults to `false`. Did you got other behavior?